next.config.js

```diff
@@ -19,9 +19,13 @@ const config = withTM({
       process.env.NEXT_PUBLIC_APP_DEPLOYMENT === "CANARY_CHECKER";
     const canaryPrefix = isCanary ? "" : "/canary";
     const LOCALHOST_ENV_URL_REWRITES = [
+      {
+        source: "/api/.ory/:path*",
+        destination: `${backendURL}/kratos/:path*`
+      },
       {
         source: "/api/:path*",
-        destination: `${backendURL}/api/:path*`
+        destination: `${backendURL}/:path*`
```

Comment:
This change needs to be carefully tested because it also impacts Netlify testing and might impact the deployed version.

Reply:
This was pushed by accident. Didn't realize that. Will remove it